src/myrtlespeech/model/rnn.py

```diff
@@ -11,6 +12,37 @@ class RNNType(IntEnum):
     BASIC_RNN = 2
 
 
+#: The type of an :py:class:`RNN` hidden state.
+#:
+#: Depending on the :py:class:`RNN`'s :py:class:`RNNType`, the hidden state
+#: will either be a length 2 Tuple of :py:class:`torch.Tensor`s or a single
```

Comment:
```suggestion
#: will either be a length 2 Tuple of :py:class:`torch.Tensor`\s or a single
```

Sphinx is rendering this as a continuous piece of code from `:py:class:...` the end of the next lines `torch.Tensor` unless the backtick is escaped.

Reply:
Oops! Thanks